docs/dictionary/function/cos.lcdoc

```diff
@@ -35,18 +35,16 @@ The <cos> <function> <return|returns> a number between -1 and 1.
 Description:
 Use the <cos> <function> to find the cosine of an angle.
 
-The <cos> <function> repeats every 2*pi <radian|radians>: cos(x) = cos(x
-
-+ 2 * pi), for any value of x.
+The <cos> <function> repeats every 2*pi <radian|radians>: cos(x) = 
+cos(x + 2 * pi), for any value of x.
```

Comment:
you might put `2*pi`, `cos(x) = +cos(x + 2 * pi)` and `x` in \`backticks\` here so they show up as code